app/services/delivery_request_service.rb

```diff
@@ -43,45 +46,36 @@ def call(email:)
 
 private
 
-  def call_provider(address, reference, email)
+  attr_reader :attempt_id, :email, :provider_name, :subject_prefix, :overrider
+
+  def address
+    @address ||= begin
+      overrider.destination_address(email.address).tap do |address|
+        next if address == email.address
```

Comment:
Fair, I'm quite used to them myself. After some juggling I decided to drop the logging behaviour as I'm not sure when any value is gained from it (or if it even works given it's a Rails logger running in a Sidekiq context).

Reply:
Ah, I did actually benefit from this logging recently, because it explained why no email was being sent: "Overriding email address foo@bar.com to <blank>". This is kind of covered by [the docs](https://docs.publishing.service.gov.uk/manual/load-test-email-alert-api.html#testing-without-notify) now, so I guess we're unlikely to get more benefit from it.

Reply:
Oh sure, that's interesting. One of my observations was that the logging doesn't capture the nil case which probably ends up looking confusing.